xtokens/src/mock/para.rs

```diff
@@ -221,6 +222,9 @@ impl cumulus_pallet_xcmp_queue::Config for Runtime {
 	type ChannelInfo = ChannelInfo;
 	type VersionWrapper = ();
 	type ExecuteOverweightOrigin = EnsureRoot<AccountId>;
+	type ControllerOrigin =
+		EnsureOneOf<EnsureRoot<AccountId>, EnsureXcm<IsMajorityOfBody<RelayLocation, ExecutiveBody>>>;
```

Comment:
`EnsureXcm<IsMajorityOfBody<RelayLocation, ExecutiveBody>>` is for relay chain council I believe. We won't need it for xtokens testing.

Reply:
use just `EnsureRoot`.